src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp

```diff
@@ -91,12 +91,12 @@ QJSValue ControllerScriptEngineLegacy::wrapFunctionCode(
 }
 
 void ControllerScriptEngineLegacy::setScriptFiles(
-        const QList<LegacyControllerMapping::ScriptFileInfo>& scripts) {
+        QList<LegacyControllerMapping::ScriptFileInfo> scripts) {
```

Comment:
I think the actually generated a clazy warning because QList is too large?

Reply:
I don't think so, because it is kept inside the object. So a copy-by value is here the preferred signature.

Reply:
clazy is not complaining. Due to the std::move() passing a const&  is no longer possible. I assume clazy also consideres that.

Reply:
Okay. Lets just merge and see then. Was just a speculation on my part, neither do I necessarily agree with clazy either.